### DISCUSSION TITLE
Bootstrap-select v1.14.0-beta2 dropdown menu is not hiding when we press enter after selecting any option.

### DISCUSSION THREAD

**Discussion by Adityageek** [snapappointments/bootstrap-select#2638] [Q&A]
Hi guys, currently I am migrating to Bootstrap-select v1.14.0-beta2 with bootstrap 5 but the dropdown menu is not hiding when we press Enter key after selecting an option, but the same is working with the click event, anyone knows the issue. Please help me with this issue.

**Comment by sangbinlee** [comment]
me too ^^
